cli/command/formatter/stats.go

```diff
@@ -139,7 +141,18 @@ type containerStatsContext struct {
 
 func (c *containerStatsContext) Container() string {
 	c.AddHeader(containerHeader)
-	return c.s.Name
+	return c.s.Container
+}
+
+func (c *containerStatsContext) Names() string {
```

Comment:
@vdemeester `docker ps` used to show all the things a container could be referenced as which includes names, as well as link aliases. That's why it is plural.... it might still do that if you use `--no-trunc` come to think of it.

Definitely singular here is good.